### ISSUE TITLE
Random(?) inability to package CBMs

### ISSUE THREAD

**Issue by PatrikLundell** [CleverRaven/Cataclysm-DDA#58999]
### Describe the bug

I've recovered a number of CBMs from zomborgs, and out of the 6 or so I've tried to "process" (wash, reset, repackage, sterilize), two fail for unknown reasons at the packaging stage (i.e. putting them into autoclave pouches) with 3-4 attempts for each of them just being met with a message that it failed.
One is a regular power CBM, and the other is my second arm armor plating one (so I've succeeded with one). There is nothing to indicate I've failed to reset the CBMs properly.

### Steps to reproduce

1. Get a bunch of CBMs from zomborgs (and probably any other suitable source).
2. Wash them.
3. Reset them.
4. Package them in autoclave pouches.
5. Have some of them consistently fail to get packaged for some reason.
6. Scratch head.

### Expected behavior

Either randomly fail to package CBMs (which is weird, given that it doesn't seem to be a particularly complicated process), or provide you with a message informing you why it failed.

### Screenshots

A third CBM that doesn't want to be prepared...
![Screenshot (248)](https://user-images.githubusercontent.com/22739822/177412637-6c191577-8894-46fe-87e7-df46585013af.png)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19043.1766 (21H1)
- Game Version: be2fdc3 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]
+ JSON "mod" to disable portals storms
+ JSON addition of Workshop welding consumable recipes.

**Comment by chrispikula** [comment]
I've noticed that it's not a guaranteed success, even with high stats and skills, although they do help.  And when it does happen, like you said, it tends to happen to the same one repeatedly.  I 'fix' this by advancing time, aka, walking around a little bit.

**Comment by PatrikLundell** [comment]
I haven't had any success with your "fix". The failures of the first two had my character walk away to the stockpile to get a new pouch before walking back and make a new attempt (2-3 world tile round trip), and I've tried old failures again much later.

However, I'll try again when I'm sure I haven't just failed a different CBM. If the failure is based on a coarse time stamp the RNG "roll" isn't generated properly, however.

**Comment by hexagonrecursion** [comment]
@PatrikLundell Did your issue get resolved? Was it just bad luck? If not, please upload your save file.

**Comment by chrispikula** [comment]
I have run into this same phenomena, and I do recall that sometimes it'd take 10+ pouches to pack a finicky CBM.  Skipping to the next on the list like Patrik mentioned was what I usually did.  It was around, hmm, about 1 in 6 or 1 in 8 cbm's having that issue.  Didn't matter if they were the working or mangled ones either.  

Yes, I was sterilizing broken cbms, don't give me a weird look.  :-p

It was slightly annoying, as you'd lose the pouches, and have to make more.  The lack of message saying something like 'you ripped the bag' was irritating.

**Comment by PatrikLundell** [comment]
Hm, weird. I copied the save, teleported home, saved, and tested. In each of the about 5 test suites I tried to pack each of my about 5 prepared and unpackaged CBMs one after the other, retrying the ones failing, and I had about 1-2 failures in the suites, with no repeated failures as I had before, so it seems to be a case of significant bad luck (or some unknown factor that messes up the RNGs only under certain conditions).

I'll make a new set of tests after returning to base "for real", but currently it looks like bad luck.
